lms/static/js/spec/learner_dashboard/entitlement_unenrollment_view_spec.js

```diff
@@ -0,0 +1,193 @@
+define([
+    'backbone',
+    'jquery',
+    'js/learner_dashboard/views/entitlement_unenrollment_view'
+], function(Backbone, $, EntitlementUnenrollmentView) {
+    'use strict';
+
+    describe('EntitlementUnenrollmentView', function() {
+        var view = null,
+            options = {
+                dashboardPath: '/dashboard',
+                signInPath: '/login'
+            },
+
+            initView = function() {
+                return new EntitlementUnenrollmentView(options);
+            },
+
+            modalHtml = '<a id="link1" class="js-entitlement-action-unenroll"                         ' +
+                        '   data-course-name="Test Course 1"                                          ' +
+                        '   data-course-number="test1"                                                ' +
+                        '   data-entitlement-api-endpoint="/test/api/endpoint/1">Unenroll</a>         ' +
+                        '<a id="link2" class="js-entitlement-action-unenroll"                         ' +
+                        '   data-course-name="Test Course 2"                                          ' +
+                        '   data-course-number="test2"                                                ' +
+                        '   data-entitlement-api-endpoint="/test/api/endpoint/2">Unenroll</a>         ' +
+                        '<div class="js-entitlement-unenrollment-modal">                              ' +
+                        '  <span class="js-entitlement-unenrollment-modal-header-text"></span>        ' +
+                        '  <span class="js-entitlement-unenrollment-modal-error-text"></span>         ' +
+                        '  <button class="js-entitlement-unenrollment-modal-submit">Unenroll</button> ' +
+                        '</div>                                                                       ';
+
+        beforeEach(function() {
+            setFixtures(modalHtml);
+            view = initView();
+        });
+
+        afterEach(function() {
+            view.remove();
+        });
+
+        describe('click .js-entitlement-action-unenroll', function() {
```

Comment:
These describe statements should be sentences, since they come up in test results as titles:

![screen shot 2017-12-13 at 4 02 50 pm](https://user-images.githubusercontent.com/3861052/33962291-195f2358-e01f-11e7-87d1-4e4f3689cca0.png)